drift.c

```diff
@@ -11,19 +11,19 @@
 #include "endrun.h"
 
 
-static int drift_particle_full(int i, int ti1, int blocking);
+static int drift_particle_full(int i, unsigned int ti1, int blocking);
```

Comment:
I think the fact the work is dirgy suggests there is benefit of doing so. I'll do this.